app/models/decidim/kids/minor_account.rb

```diff
@@ -16,6 +16,13 @@ class MinorAccount < ApplicationRecord
       validate :can_be_tutor
       validate :can_be_minor
 
+      def promote_account
+        return if minor.minor_age < (minor&.organization&.maximum_minor_age || Decidim::Kids.maximum_minor_age)
```

Comment:
```suggestion
        return if minor.minor_age <= (minor&.organization&.maximum_minor_age || Decidim::Kids.maximum_minor_age)
```
If the minor matches the maximum_minor_age is still a minor right?

Reply:
Done